extensions-core/datasketches/src/test/java/org/apache/druid/query/aggregation/datasketches/tuple/ArrayOfDoublesSketchAggregatorFactoryTest.java

```diff
@@ -56,7 +56,10 @@ public void makeAggregateCombiner()
     sketch2.update("b", new double[] {1});
     sketch2.update("c", new double[] {1});
 
-    TestObjectColumnSelector<ArrayOfDoublesSketch> selector = new TestObjectColumnSelector<ArrayOfDoublesSketch>(new ArrayOfDoublesSketch[] {sketch1, sketch2});
+    TestObjectColumnSelector<ArrayOfDoublesSketch> selector = new TestObjectColumnSelector<>(new ArrayOfDoublesSketch[]{
+        sketch1,
+        sketch2
+    });
```

Comment:
Nit: Putting the whole argument in a new line makes it more readable.
```suggestion
    TestObjectColumnSelector<ArrayOfDoublesSketch> selector = new TestObjectColumnSelector<>(
        new ArrayOfDoublesSketch[]{sketch1, sketch2}
     );
```